NitroxClient/Communication/Packets/Processors/CyclopsActivateHornProcessor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Packets;
+using System;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class CyclopsActivateHornProcessor : ClientPacketProcessor<CyclopsActivateHorn>
+    {
+        private PacketSender packetSender;
+
+        public CyclopsActivateHornProcessor(PacketSender packetSender)
+        {
+            this.packetSender = packetSender;
+        }
+
+        public override void Process(CyclopsActivateHorn hornPacket)
```

Comment:
IIRC we use the variable name `packet` in most if not all other packet processors, for consistency.

Reply:
In CyclopsToggleFloodlightsProcessor the name is lightingPacket...

Reply:
You are right (And my IIRC is wrong :wink:). About 50% is named `packet` (because that is what VS names it when auto-implementing the abstract class), and the other 50% has a name related to the processor.

Reply:
Ok and what should I do now (and what means IIRC) ?

Reply:
`If I Remember Correctly` - I wasn't completely sure in the first place.

You can just ignore this, it's very very minor anyway.